sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -38,6 +39,10 @@
     <PackageReference Include="System.Net.Http" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
+    <Reference Include="System.Net.Http" />
```

Comment:
Can you share more about how including System.Net.Http as a reference in the net472 target prevents it from pulling in the transitive dependencies this target brings?

Reply:
The PackageReference to `System.Net.Http` only exists for the net461 TFM but not for the net472 TFM as .NET Framework 4.7.2 includes the necessary API inbox (as part of the framework). The System.Net.Http package is undesirable as a (transitive) package reference as the package is quite old, not serviced anymore and brings in even more transitive dependencies that aren't serviced anymore.

Reply:
Thank you!  I had not understood the distinction between `<PackageReference/>` and `<Reference/>`.